### DISCUSSION TITLE
Can't get filesize for released file

### DISCUSSION THREAD

**Discussion by N6REJ** [community/community#134310] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I've tried everything I can think of and it always gives me a content-length of 0.  How can I get just the file size of a given file like 
https://github.com/Bearsampp/module-php/releases/download/2024.7.11/bearsampp-php-7.4.30-2024.7.12.7z

**Comment by oussemamansouri** [comment]
Can you explain more?

**Comment by N6REJ** [comment]
it doesn't matter if I use curl or file_get_contents or fopen the header always says the content-length is zero so I have no way to get the proper filesize that I can figure out.   I'm trying to make a progress bar
